grails-forge-core/src/main/java/org/grails/forge/feature/test/Geb.java

```diff
@@ -105,24 +105,29 @@ public void apply(GeneratorContext generatorContext) {
                 .artifactId("geb")
                 .test());
 
+        generatorContext.addDependency(Dependency.builder()
+                .groupId("org.gebish")
+                .artifactId("geb-spock")
+                .test());
```

Comment:
This is already a transient dependency from `testFixtures('org.grails.plugins:geb')`

Reply:
integrationTestImplementation testFixtures('org.grails.plugins:geb' allowed this to be removed